### PR TITLE
Simplify `is_systemd_used`

### PR DESCRIPTION
**Pull request by danielhuang**
Code to detect if `/run/systemd/system` file exists

### CODE REVIEW

File: src/systemd.rs [hunk 1]
```diff
@@ -33,21 +33,7 @@ pub fn stop_systemd_target() {
...
-					false
-				}
-			}
-			Err(error) => {
-				warn!("unable to check if systemd is used: {}", error);
-				false
-			}
-		}
-	})
+	IS_SYSTEMD_USED.get_or_init(|| std::fs::metadata("/run/systemd/system").is_ok())
```
Comment (Drakulix) [inline src/systemd.rs:36]: Why metadata instead of `std::path::Path::exists`?

Comment (danielhuang) [reply src/systemd.rs:36]: The functionality is equivalent: https://doc.rust-lang.org/stable/src/std/path.rs.html#2872-2874

Comment (Drakulix) [reply src/systemd.rs:36]: That is an implementation detail of `std` though. It may or may not keep being the same, we should use the simpler method.

Comment (danielhuang) [reply src/systemd.rs]: Done.

**Review by Drakulix — APPROVED** [review]
Thanks :)

**Pull request merged**